### ISSUE TITLE
[Bug Report] VTextInput clearable icon doesn't have aria-label

### ISSUE THREAD

**Issue by dawidstefaniak** [vuetifyjs/vuetify#10370]
### Environment
**Vuetify Version:** 2.2.7
**Vue Version:** 2.6.11
**Browsers:** Opera 66.0.3515.44, Google Chrome
**OS:** Mac OS 10.15.2

### Steps to reproduce
1. Create `v-text-field`
2. Set `clearable` parameter to `true`

### Expected Behavior
The `append-inner-icon` when `clearable` is set to `true` does not have `aria-label`, what reduces SEO Accessibility ranking.

### Actual Behavior
The `append-inner-icon` when clearable is set to `true` should have `aria-label`

### Reproduction Link
<a href="https://codepen.io/dawidstefaniak/pen/KKwLdrd" target="_blank">https://codepen.io/dawidstefaniak/pen/KKwLdrd</a>

### Other comments
I think it will be okay to set hard-coded `aria-label` for the `clearable` icon.

**Comment by jmatth** [comment]
Piggybacking off of this: it might also be good to set `aria-hidden="true"` on the clearable icon while the box is empty. Currently screen readers will pick up the button even though it is visually hidden and clicking it is pointless if the text box is already empty.

**Comment by mnannola** [comment]
Currently we have some code in our apps that manually add the aria-label attribute to the clearable icons, but we would much rather have a built in solution to add the label.

**Comment by KaelWD** [comment]
This was resolved in v2.2.9 with #10439
